infrastructure/simulation-tool/test/operations/deposit.test.ts

```diff
@@ -0,0 +1,81 @@
+import { Wallet as RollupWallet, RootstockOperation } from '@rsksmart/rif-rollup-js-sdk';
+import { PriorityOperationReceipt } from '@rsksmart/rif-rollup-js-sdk/build/types';
+import { expect, use } from 'chai';
+import { ContractReceipt, Wallet as EthersWallet, constants } from 'ethers';
+import sinon from 'sinon';
+import sinonChai from 'sinon-chai';
+import config from '../../src/utils/config.utils';
+import { executeDeposit, getDepositResult, prepareDeposit } from '../../src/operations/deposit';
+
+use(sinonChai);
+
+describe('prepareDeposit', () => {
+    it('should return a deposit parmenters with amount between min and max', () => {
+        const l2sender = sinon.createStubInstance(RollupWallet);
+        const l1recipient = sinon.createStubInstance(EthersWallet);
+        const deposit = prepareDeposit(l2sender, l1recipient);
+        const [minAmount, maxAmount] = config.weiLimits.deposit;
+
+        expect(deposit.amount.gte(minAmount)).to.be.true;
+        expect(deposit.amount.lt(maxAmount)).to.be.true;
+    });
+    it('should return a deposit with token address set to zero', () => {
+        const l2sender = sinon.createStubInstance(RollupWallet);
+        const l1recipient = sinon.createStubInstance(EthersWallet);
+        const deposit = prepareDeposit(l2sender, l1recipient);
+
+        expect(deposit.token).to.eq(constants.AddressZero);
+    });
+    it('should return a deposit with depositTo set to l1recipient address', () => {
+        const l2sender = sinon.createStubInstance(RollupWallet);
+        const l1recipient = sinon.createStubInstance(EthersWallet);
+        const deposit = prepareDeposit(l2sender, l1recipient);
+
+        expect(deposit.depositTo).to.eq(l1recipient.address);
+    });
+});
+
+describe('executeDeposit', () => {
```

Comment:
Why don't we have something like `should execute all deposits`?

Reply:
no reason

Reply:
added

Reply:
I think you've just added `should execute all transfers` 😂

Reply:
changed